values.yaml

```diff
@@ -111,8 +111,19 @@ postgresql:
           );
           CREATE TABLE submission (
             id SERIAL PRIMARY KEY,
-            student_id INTEGER REFERENCES student (id),
+            student_id INTEGER REFERENCES student (id) NOT NULL,
+            assignment_id INTEGER REFERENCES assignment (id) NOT NULL,
```

Comment:
Won't you need to put the CREATE TABLE for the `assignment` table before the `submission` table now, because of this foreign key? I would think it would fail if it tries to create the `submission` table and can't find the `assignment` table.

Reply:
Yeah, I didn't consider the order, it will probably fail. And yes, it should be develop lol